classes.py

```diff
@@ -2,8 +2,16 @@
 #-*- coding: utf-8 -*-
 
 import collections
+import requests
+import re
+
+
+TIMETABLE_FETCH_URL = 'https://hercules-10496.herokuapp.com/api/v1/faculty/timetable?name={name}&dept={dept}'
```

Comment:
You can use query parameters in `requests`. 
Generally, it is not advised to use string formatting while constructing URLs.